spec/presenters/coronavirus/sub_section_json_presenter_spec.rb

```diff
@@ -44,16 +44,21 @@
       expect(subject.output).to eq(expected)
     end
 
-    context "with featured links" do
-      it "looks up the description in Publishing API for a relative featured link" do
-        sub_section.featured_link = path
-        description = Faker::Lorem.sentence
+    context "with action link url" do
+      it "looks up the description in Publishing API for a relative action link url" do
```

Comment:
I don't think this test description is correct anymore, I don't think we do any looking up now.

Reply:
This test has been removed as once I removed the stubs for publishing-api I realised this test was testing anything new.